azurePipelineTemplates/run_scenarios.yml

```diff
@@ -0,0 +1,189 @@
+parameters:
+  - name: storage_account_name
+    type: string
+  - name: container_name
+    type: string
+  - name: orp_source_account_name
+    type: string
+  - name: orp_destination_account_name
+    type: string
+  - name: orp_source_container_name
+    type: string
+  - name: destination_container_name
+    type: string
+  - name: goos
+    type: string
+  - name: goarch
+    type: string
+  - name: cgo_enabled
+    type: string
+  - name: build_name
+    type: string
+  - name: azcopy_msi_app_id
+    type: string
+
+steps:
```

Comment:
Is there some reason why this can't be done within the E2E framework? If so, this'd be a really valuable bit of feedback on it.

Reply:
More specifically, understanding what makes using that difficult is the aim. I have marked this PR as requested changes, in case it is feasible. If it is not, I'll unblock.

Reply:
There are two test cases added as part of this Additional E2E 
1. Test end-to-end functionality for copying data between ORP source and destination accounts, noting that ORP accounts are not currently supported in the E2E framework.

2. Verify that older versions of AzCopy do not append version messages to piped downloads by comparing output files from older and newer versions to ensure consistency fow which we are downloading the older version and executing the command.

Reply:
These are both absolutely possible within the E2E test framework. Especially number 2 would not require any additional static accounts. I'll let this go since the work is done, but please prefer to implement things in the E2E framework in the future.

Just because it doesn't support something now, doesn't mean it's outside of your range to try to implement, nor does it mean we should work around it. We own 100% of that code. If something doesn't work yet in it, we can implement it.